tox.ini

```diff
@@ -42,15 +41,13 @@ commands =
     --cov-report=xml
 
 [testenv:py38-docs]
-changedir = doc
 extras = docs
 commands =
-    # remove autosummary output
-    rm -rf {toxinidir}/doc/source/_neurom_build
-    make clean
-    make html SPHINXOPTS=-W
+    # remove autosummary output and cleanup
+    rm -rf {toxinidir}/doc/source/_neurom_build {toxinidir}/doc/build/*
+    sphinx-build -W -b doctest {toxinidir}/doc/source {toxinidir}/doc/build/doctest
+    sphinx-build -W -b html {toxinidir}/doc/source {toxinidir}/doc/build/html
```

Comment:
@GianlucaFicarelli , using the cli directly solves all the path problems and removes all the make files. What do you think?

Reply:
@mgeplf, I would like your opinion too on this one. :)

Reply:
Will do.

Reply:
Ok, I'm happy with removing the makefile and going this way.